pages/api/upload_album.py

```
@@ -1238,3 +1262,69 @@ def get_folder_info_by_name():
+        return send_from_directory('storage', filename)
+    except Exception as e:
+        print(f"Error serving file: {str(e)}")
+        return "File not found", 404
+
+# Lấy đường dẫn tuyệt đối của project
+BASE_DIR = Path(__file__).resolve().parent.parent.parent
+STORAGE_PATH = BASE_DIR / 'storage'
+PUBLIC_PATH = BASE_DIR / 'public'
+
+@app.route('/static/icon_folder/<path:filename>')
+def serve_icon(filename):
```

Ruff (0.7.0)

Redefinition of unused `serve_icon` from line 905

(F811)